raphtory-graphql/src/model/graph/graphs.rs

```diff
@@ -0,0 +1,48 @@
+use crate::model::graph::property::GqlProperties;
+use dynamic_graphql::{ResolvedObject, ResolvedObjectFields};
+use itertools::Itertools;
+use raphtory::{
+    db::api::{properties::dyn_props::DynProperties, view::DynamicGraph},
+    prelude::GraphViewOps,
+    search::{into_indexed::DynamicIndexedGraph, IndexedGraph},
+};
+
+#[derive(ResolvedObject)]
+pub(crate) struct GqlGraphs {
+    graphs: Vec<IndexedGraph<DynamicGraph>>,
+}
+
+impl GqlGraphs {
+    pub fn new<G: DynamicIndexedGraph>(graphs: Vec<G>) -> Self {
+        Self {
+            graphs: graphs
+                .into_iter()
+                .map(|g| g.into_dynamic_indexed())
+                .collect(),
+        }
+    }
+}
+
+#[ResolvedObjectFields]
+impl GqlGraphs {
+    async fn names(&self) -> Vec<String> {
+        self.graphs
+            .iter()
+            .map(|g| g.properties().constant().get("name").unwrap().to_string())
+            .collect()
+    }
+
+    async fn properties(&self) -> Vec<GqlProperties> {
```

Comment:
As much as I would love these in - I don't think we can do them with the bin graphs if we have to load many graphs off disk